### PR TITLE
Features/fixes

### PR DESCRIPTION
**Pull request by MennoVK**

### CODE REVIEW

File: static/popup.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <button id="options-page">Change colors</button>
     <footer>
       Created by <a href="https://github.com/Pachciar3/">Dawid Pachciarek</a> &
-      <a href="https://github.com/MennoVK/">Menno Veerkamp</a>
```
Comment (Pachciar3) [inline static/popup.html:18]: Hi, I think you forgot to change the link on the options.html page ;-)

Comment (MennoVK) [reply static/popup.html:18]: Ah you are right. Is the way i made changes this time correct btw?

Comment (Pachciar3) [reply static/popup.html:18]: Almost good :-) 
1. When you make the pull request, you should assign me as the reviewer. When you do, I get an email about a new pull request that needs my review.
2. Good practice: You should write commit messages in the present tense. 

Generally do not worry too much about it. The most important thing is that someone takes care of this project 🥇.

Comment (MennoVK) [reply static/popup.html:18]: Ah interesting, thanks!

Comment (Pachciar3) [reply static/popup.html:18]: Hi, will you be adding changes to `options.html` ?

Comment (MennoVK) [reply static/popup.html:18]: I was very busy, sorry. Should be fixed now!

**Review by Pachciar3 — APPROVED** [review]
Good work !

**Pull request merged**